frontend/packages/dev-console/integration-tests/support/step-definitions/common/addFlow.ts

```diff
@@ -57,10 +57,6 @@ When('user clicks Instantiate Template button on side bar', () => {
   catalogPage.clickButtonOnCatalogPageSidePane();
 });
 
-Given('user is at Developer Catalog page', () => {
-  addPage.selectCardFromOptions(addOptions.DeveloperCatalog);
-});
-
```

Comment:
why this statement is removed?

Reply:
Since this is there in master. please ignore this.